server/filters.go

```diff
@@ -225,6 +225,51 @@ func defaultFilter(versions map[string]*pbVersion.Version, apply string) error {
 	return deleteOtherBut(apply, versions)
 }
 
+func pgDepFilter(versions map[string]interface{}, productVersion string) (string, error) {
+	if len(versions) == 0 {
+		return "", nil
+	}
+
+	keys := make([]string, 0, len(versions))
+	for k := range versions {
+		keys = append(keys, k)
+	}
+
+	sorted, err := sortedVersionsDesc(keys)
+	if err != nil {
+		return "", status.Errorf(codes.Internal, "failed to sort versions: %v", err)
+	}
+
+	desired := sorted[0].String()
+	for _, s := range sorted {
+		versStr := s.String()
+		if strings.HasSuffix(versStr, ".0") {
+			versStr = strings.TrimSuffix(versStr, ".0")
```

Comment:
operator version can be any, it does not always ends with `.0`.
can we avoid cutting `.0`?

Reply:
it is about postgresql version, And we cut '.0' only  after we get 'version' var from 'semvar' package witch  always add '.0', so we have '13.4.0' and  so on